dataloader.py

```diff
@@ -10,7 +10,7 @@
 from tensorflow_federated.python.common_libs import py_typecheck
 
 
-def get_client_data(dataset_name, mask_by, mask_ratios, sample_client_data=False):
+def get_client_data(dataset_name, mask_by, mask_ratios, sample_client_data=False, only_digits=True):
```

Comment:
I don't see the need to add the only digits parameter since I dont think the idea of using more than digits came up in discussions. If it becomes necessary to add the parameter, i'd recommend adding it to the config file.

Reply:
Sure, but I think it helps to be explicit in the load_data function.